src/sbt-test/dumpLicenseReport/custom-report/test

```diff
@@ -1,4 +1,5 @@
 > dumpLicenseReport
 $ exists target/license-reports/lreport.md
 > check
-$ exists target/license-reports/test-config.html
\ No newline at end of file
+$ exists target/license-reports/test-config.html
```

Comment:
I believe that this changes the tested expectation from "`target/license-reports/test-config.html` does not exist" to "`exists target/license-reports/test-config.html` exists". Was that intentional?